wordpress-mobile/WordPress-iOS
# My Site Dashboard: Fix truncated title on next post card

Fixes #18266

## Description
Enabled auto shrinking for the title label in the next post view card.

<img src="https://user-images.githubusercontent.com/25306722/161673253-d45bd6dd-0959-47af-95b5-116c2b56f935.png" width=300>

## Testing Instructions

1. Install the app on a small screen device (iPhone 8)
2. Choose a site that has published posts, and has no drafts or scheduled posts.
3. Open the dashboard
4. Make sure the title "Create your next post" is not truncated.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
